README_B2C.md

```diff
@@ -0,0 +1,167 @@
+---
+page_type: sample
+languages:
+- python
+- html
+products:
+- azure-active-directory
+description: "This sample demonstrates a Python web application calling a Microsoft Graph that is secured using Azure Active Directory."
```

Comment:
>a [](start = 72, length = 1)

nit: remove "a"

Reply:
Actually I realize that in this sample, the app dev is not calling Microsoft Graph. He/she is calling another web api which has been previously deployed to their B2C tenant. So, how about this?
"... calling another web api in your B2C tenant that is secured using ..."